language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/testgen/AnnotationConfigsProcessor.java

```diff
@@ -0,0 +1,145 @@
+package org.ballerinalang.langserver.command.testgen;
+
+import org.wso2.ballerinalang.compiler.tree.BLangAnnotationAttachment;
+import org.wso2.ballerinalang.compiler.tree.expressions.BLangArrayLiteral;
+import org.wso2.ballerinalang.compiler.tree.expressions.BLangExpression;
+import org.wso2.ballerinalang.compiler.tree.expressions.BLangLiteral;
+import org.wso2.ballerinalang.compiler.tree.expressions.BLangRecordLiteral;
+import org.wso2.ballerinalang.compiler.tree.expressions.BLangSimpleVarRef;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.BiConsumer;
+
+/**
+ * This class provides capabilities for processing annotation configurations.
```

Comment:
Missing version

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/commit/481d65eb44aebd87a81bcd56982b6a3eb1bbd9b6